app/views/home/how_government_works.html.erb

```diff
@@ -1,534 +1,489 @@
+        <p class="govuk-body"><%= link_to "See statistics publications on GOV.UK", "/search/research-and-statistics", class: "govuk-link" %></p>
 
-            <p class="govuk-body"><%= link_to "See statistics publications on GOV.UK", "/search/research-and-statistics", class: "govuk-link" %></p>
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Transparency",
+          font_size: "m",
+          heading_level: 3,
+          margin_bottom: 4,
+        } %>
 
-          </div>
-        </div>
-        <div class="one-third">
-          <div class="inner-block">
-            <%= render "govuk_publishing_components/components/heading", {
-              text: "Transparency",
-              heading_level: 4,
-              margin_bottom: 2,
-              font_size: 19,
-            } %>
-
-            <p class="govuk-body">The government publishes information about how government works to allow you to make politicians, public services and public organisations more accountable. We are committed to publishing information about:</p>
-
-            <ul class="govuk-list govuk-list--bullet">
-              <li>how much public money has been spent on what</li>
-              <li>the job titles of senior civil servants and how much they are paid</li>
-              <li>how the government is doing against its objectives</li>
-            </ul>
-
-            <p class="govuk-body"><%= link_to 'See transparency releases on GOV.UK', publications_filter_path(:publication_filter_option => 'transparency-data'), class: "govuk-link" %></p>
-
-            <%= render "govuk_publishing_components/components/heading", {
-              text: "Data",
-              heading_level: 4,
-              margin_bott2m: 4,
-              font_size: 19,
-            } %>
-
-            <p class="govuk-body">Putting data in people’s hands can help them have more of a say in the reform of public services. On <%= link_to "data.gov.uk", "https://data.gov.uk/", class: "govuk-link", rel: "external" %> you can easily find, review and use information about our country and communities - for example, to develop web applications.</p>
-          </div>
-        </div>
-      </div>
+        <p class="govuk-body">The government publishes information about how government works to allow you to make politicians, public services and public organisations more accountable. We are committed to publishing information about:</p>
 
-      <div class="thirds-width-section gov-types">
-        <div class="one-third">
-          <div class="inner-block">
-            <%= render "govuk_publishing_components/components/heading", {
-              text: "Devolved government",
-              heading_level: 3,
-              margin_bottom: 5,
-            } %>
+        <ul class="govuk-list govuk-list--bullet govuk-list--spaced">
```

Comment:
This looks like it could be swapped to use [List Component ](https://components.publishing.service.gov.uk/component-guide/list)